hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -119,14 +119,64 @@ On earlier models the serial number is used as the prefix, however, on Pi4 the M
 * 0 - Use the serial number e.g. "9ffefdef/"
 * 1 - Use the string specified by TFTP_PREFIX_STR
 * 2 - Use the MAC address e.g. "DC-A6-32-01-36-C2/"
-Default: 0
+Default: 0  
 Version: stable/pieeprom-2020-01-17.bin  
 
 ### TFTP_PREFIX_STR
 Specify the custom directory prefix string used when TFTP_PREFIX is set to 1. For example:- TFTP_PREFIX_STR=tftp_test/
-Default: ""
+Default: ""  
 Version: stable/pieeprom-2020-01-17.bin  
 
+### PXE_OPTION43
+Overrides the PXE Option43 match string with a different string. It's normally better to apply customizations to the DHCP server than change the client behaviour but this option is provided in case that's not possible.
```

Comment:
UK spelling customisations with an `s`? Dunno what the style guide says on this.

Reply:
https://github.com/raspberrypi/documentation/search?q=colour has more matches than https://github.com/raspberrypi/documentation/search?q=color :wink:

Reply:
Changed it to UK spelling, not that I care either way